src/main/kotlin/io/github/mojira/arisa/modules/KeepPrivateModule.kt

```diff
@@ -24,7 +24,15 @@ class KeepPrivateModule(
 
             val markedTime = comments.first(::isKeepPrivateTag).created
             val changedTime = changeLog.lastOrNull(::isSecurityChange)?.created
-            if (changedTime != null && changedTime.isAfter(markedTime)) {
+            val securityChange = changeLog
+                .lastOrNull() { isSecurityChangeToPublic(it, securityLevel, project.privateSecurity) }
+            if (
```

Comment:
Shouldn't `isSecurityChangeToPublic` apply to staff and non-staff actions, if so, could possibly combine this with calculating `val changedTime`.
Also, shouldn't for staff action the requirement `changedTime != null && changedTime.isAfter(markedTime)` apply as well?

Reply:
Still didn't solve the build failing though, and I have no idea why it's failing either.

Reply:
Yeah, my comment was a general recommendation and not related to the test failure.
Though maybe it would be good anyways to remove the `isSecurityChange` check and only use `securityChange` to determine the `changedTime`. Otherwise there are two different calls which determine the last relevant changelog entry, which might be error-prone and could cause issues in the future when they (for some reason) select different changelog entries.